### ISSUE TITLE
Restrict access to export based on usergroup

### ISSUE THREAD

**Issue by lairdshaw** [mybbgroup/Export-Posts#1]
As suggested by WallBB here:

https://community.mybb.com/thread-230773-post-1360687.html#pid1360687

**Comment by dragonexpert** [comment]
My thoughts on this is that users do have rights to their site data so I'm not so sure restricting it is a good idea.

**Comment by lairdshaw** [comment]
Fair point. I'm trying of think of possibly valid reasons for restricting one group and not others, but I can't come up with much. Maybe some members are wasting server resources by repeatedly hitting the export tool - but then better would be to ban them outright. Or maybe the board owner wants to use the export feature as a reward for users who achieve some goal. That would be a kind of unusual reward, and as you say, it might not be valid to withhold the "reward" from other members.

Is that for us to decide though, or for the board owner? Maybe not being able to think of a really good valid reason is just lack of imagination on my part.
